src/components/Navbar/DesktopMenu.js

```diff
@@ -116,21 +116,23 @@ function DesktopMenu() {
         </>
       ) : null}
 
-      <IconButton
+      <Button
         aria-label="show languages"
         aria-controls={languageMenuId}
         aria-haspopup="true"
         onClick={handleLanguageMenuOpen}
-        color="inherit"
+        startIcon={<TranslateIcon />}
+        endIcon={<ExpandMoreIcon />}
       >
-        <LanguageIcon />
-      </IconButton>
+        {locale[1]}
```

Comment:
This can be converted to a constant
```suggestion
        {locale[DEFAULT_LOCALE]}
```

Reply:
`1` is not actually the default locale here. 
Refer to -
```js
// src/i18n/locales.js
const LOCALES = {
  ENGLISH: ['en-us', 'English'],
  KANNADA: ['kn', 'ಕನ್ನಡ'],
};

export const localList = Object.values(LOCALES);

export default LOCALES;
```

So, `1` in this line refers to the second element of the Default locale's name itself. Similarly, `0` refers to the locale key. 

This is important because we are showing the names of the language in that language itself. 

Does that clarify?

Reply:
The fact that I had to explain means, code isn't readable. 

Maybe it could be `LOCAL_NAME_INDEX` 🤔

```js
{locale[LOCAL_NAME_INDEX]}
```

Reply:
Or it could be an object 🤔

```js
// src/i18n/locales.js
const LOCALES = {
  ENGLISH: {
    languageKey: 'en-us',
    languageName: 'English'
  },
  KANNADA: {
    languageKey: 'kn',
    languageName: 'ಕನ್ನಡ'
  },
};

export const localList = Object.values(LOCALES);

export default LOCALES;
```

Reply:
You have suggested two changes, I think the second one would be the most feasible one.

Reply:
Yup. The second one is better. Let's go with that. 👍

Updating now.